packages/documentation-site/package.json

```diff
@@ -26,6 +26,7 @@
     "@patternfly/react-user-feedback": "6.0.0-prerelease.1",
     "@patternfly/react-component-groups": "6.0.0-prerelease.3",
     "@patternfly/react-virtualized-extension": "6.0.0-prerelease.3",
+    "@patternfly/quickstarts": "6.0.0-alpha.5",
```

Comment:
Did you want to pull in version `6.0.0-prerelease.1` of quickstarts? That'll include the updated design guidelines that Erin just finished.